### PR TITLE
support custom component image.

### PR DESCRIPTION
**Pull request by morvencao**

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -218,16 +218,13 @@ func ReplaceImage(annotations map[string]string, imageRepo, componentName string
 			annotationImageRepo = DefaultImgRepository
 		}
 		// This is for test only. e.g.:
-		// if there is "mco-metrics_collector-tag" defined in annotation, use it for testing
-		componentTagSuffix, hasComponentTagSuffix := annotations["mco-"+componentName+"-tag"]
+		// if there is "mco-metrics_collector-image" defined in annotation, use it for testing
+		componentImageSuffix, hasComponentImageSuffix := annotations["mco-"+componentName+"-image"]
```
Comment (clyang82) [inline pkg/config/config.go:222]: s/componentImageSuffix/componentImage
s/hasComponentImageSuffix/hasComponentImage

Comment (morvencao) [reply pkg/config/config.go]: done

**Comment by morvencao** [comment]
Looks like I need to update the UT, fixing it....

**Comment by clyang82** [comment]
merge this pull request to accelerate the test for other components.

**Pull request merged**